projects/kalidea/kaligraphi/src/lib/02-form/kal-select-virtual-scroll/kal-select-virtual-scroll.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="kal-select-virtual-scroll__trigger" [id]="id">
+  <div (click)="open()" (focus)="open()">
+    <input
+      class="kal-select-virtual-scroll__input"
+      type="text"
+      [formControl]="searchControl"
+      placeholder="{{ label }}"
+    />
+  </div>
+
+  <ng-template cdkPortal #optionsPortal="cdkPortal">
+    <cdk-virtual-scroll-viewport
+      [itemSize]="virtualScrollConfig.itemSize"
+      class="kal-select__overlay"
+      [style.height.px]="virtualScrollHeight"
+    >
+      <div *ngIf="!options.length" class="kal-select-virtual-scroll__option">
+        No results found...
```

Comment:
We should be able to change the message. I don't know if it's possible to work with the i18n